components/org.wso2.carbon.identity.scim2.common/src/test/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManagerTest.java

```diff
@@ -2369,16 +2379,27 @@ public void testUpdateUserClaims_AddDeleteModifyClaims() throws Exception {
         doNothing().when(mockedUserStoreManager).setUserClaimValuesWithID(any(), anyMap(), isNull());
         doNothing().when(mockedUserStoreManager).deleteUserClaimValueWithID(any(), any(), any());
 
-        Method m = SCIMUserManager.class.getDeclaredMethod(
-                "updateUserClaims",
-                User.class, Map.class, Map.class, Map.class
-        );
-        m.setAccessible(true);
-        m.invoke(scimUserManager, user, oldClaims, newClaims, multiValuedClaims);
+        scimCommonComponentHolder.when(SCIMCommonComponentHolder::getClaimManagementService)
+                .thenReturn(mockClaimMetadataManagementService);
+
+        IdentityEventService mockService = mock(IdentityEventService.class);
+        scimCommonComponentHolder.when(SCIMCommonComponentHolder::getIdentityEventService).thenReturn(mockService);
+
+        try (MockedStatic<org.wso2.carbon.identity.application.authentication.framework.util.FrameworkUtils> fw =
+                     mockStatic(org.wso2.carbon.identity.application.authentication.framework.util.FrameworkUtils.class)) {
+            fw.when(FrameworkUtils::getMultiAttributeSeparator).thenReturn(",");
+
+            Method m = SCIMUserManager.class.getDeclaredMethod(
+                    "updateUserClaims",
+                    User.class, Map.class, Map.class, Map.class
+                                                              );
+            m.setAccessible(true);
+            m.invoke(scimUserManager, user, oldClaims, newClaims, multiValuedClaims);
```

Comment:
Do these changes in the tests related to the changes that we are doing in the PR ?